srcpkgs/vala-panel/template

```diff
@@ -1,20 +1,25 @@
 # Template file for 'vala-panel'
 pkgname=vala-panel
-version=0.4.88
+version=0.5.0
 revision=1
-build_style=cmake
-hostmakedepends="gettext pkg-config vala glib-devel"
-makedepends="vala-devel cmake-vala gtk+3-devel"
-short_desc="GTK+3 desktop panel inspired by LXPanel"
-maintainer="John Eira <john.eira@web.de>"
+build_style=meson
+configure_args="$(vopt_if wayland -Dplatforms=x11,layer-shell -Dplatforms=x11)"
+hostmakedepends="vala pkg-config cmake gettext"
```

Comment:
Re-add glib-devel in host and I think that will fix the failing builds.

Reply:
Thanks, you are right, moving glib-devel to hostmakedepends fixed it.